lib/robots/dor_repo/gis_assembly/generate_structural.rb

```diff
@@ -110,160 +95,96 @@ def file_access_params
         end
 
         def data_files_params
+          params = GisRobotSuite.locate_data_files(content_dir).map do |file|
```

Comment:
This is the primary addition, whereby the data files are added to the structural.